scss/abstracts/_variables.scss

```diff
@@ -0,0 +1,14 @@
+$primary-color: #754ffe;
+$primary-dark: #593cc1;
+$primary-light: #e3dcff;
+$secondary-color: #6343d8;
+$white-default: #ffffff;
+$black-default: #000000;
+$box-shadow-color: #cccccc;
+$item-grey: #64748b;
+$bg-hover: #f1f5f9;
+$border-color: #94a3b8;
+$border-bottom-color: #cbd5e1;
+$secondary-btn-color: #334155;
+$secondary-btn-hover: #f1f5f9;
+$star-color: rgb(245, 158, 11);
```

Comment:
Try to unify units format - if you used hex notation, use it for each color - if you want to use rgb() it's also ok, but leave hex notation then

Reply:
You're right, I'll change that